doc/source/general/compiling.rst

```diff
@@ -18,7 +18,33 @@ When you try to run your script, the first thing that kOS does is transform your
 
 This process of transforming your script into Machine Language, or "ML" is called "Compiling".
 
-The ":ref:`RUN`" and ":ref:`RUNPATH`" commands do this silently, without telling you. This is why you may have noticed the universe slightly stutter for a moment when you first run your program. Compiling is hard work for the universe to do.
+The :key:`RUN` and :func:`RUNPATH` commands do this silently, without telling
+you.  The :key:`COMPILE` command explicitly compiles the file and saves it for
+future use.
+
+.. note::
+    .. versionchanged:: 1.1.0
+        kOS now performs compilation on a parallel thread (thought to run in a
+        parallel universe).  This means that the universe no longer needs to
+        freeze in order to perform the compilation.  As with the execution of
+        other triggers, compiling from mainline code will not interupt or pause
+        triggers but compiling within a trigger will result in both the mainline
+        and trigger code pausing until the compilation is complete.
```

Comment:
Hm, I see what you're saying.  It never explicitly states that the mainline code is paused during the compilation process.  Let me think about that a little bit to see if I can restructure to make a short but clear description.